kwasm-lemmas.md

```diff
@@ -72,6 +78,10 @@ Since 0 <= x mod m < m <= n, (x mod m) mod n = x mod m
        andBool N >Int 0
        andBool M modInt N ==Int 0
       [simplification]
+
+    rule #wrap(N, #wrap(M, X)) => #wrap(N, X)
+      requires notBool (M <=Int N)
+      [simplification]
 ```
```

Comment:
Let's keep the `#wrap` lemmas together in one place, may even consider making them one-liners and lining up the `requires` and `[simplificatio]`.

Reply:
hmmm, NVM, it seems there are lots of places where there are lemas for `#wrap`

Reply:
My thinking here was that the `#wrap`s are special cases of `modInt`s, and should be grouped with the corresponding `modInt` rule. With our new, total, definition of `#wrap` they are slightly different, but still similar enough I think we should group them. Because if a `modInt` rule is sound, there should be an obvious `#wrap` rule that also is sound.